scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamMode.scala

```diff
@@ -0,0 +1,79 @@
+package com.twitter.scalding.beam_backend
+
+import com.twitter.scalding.Execution.Writer
+import com.twitter.scalding.typed.{Resolver, TypedSink, TypedSource}
+import com.twitter.scalding.{Config, Mode, TextLine}
+import org.apache.beam.sdk.Pipeline
+import org.apache.beam.sdk.io.TextIO
+import org.apache.beam.sdk.values.PCollection
+import scala.concurrent.ExecutionContext
+
+case class BeamMode(
+  pipeline: Pipeline,
```

Comment:
This can't be correct, because this means you keep running same `pipeline` again and again with every `execute()` call